spec/app/github_repo_fetcher_spec.rb

```diff
@@ -1,18 +1,29 @@
 RSpec.describe GitHubRepoFetcher do
   describe "#repo" do
-    it "returns a repo if the user is specified" do
-      stub_request(:get, "https://api.github.com/repos/some-user/some-repo")
-        .to_return(body: "{}", headers: { content_type: "application/json" })
+    before do
+      stub_request(:get, "https://api.github.com/users/alphagov/repos?per_page=100")
```

Comment:
```suggestion
      stub_request(:get, "https://api.github.com/orgs/alphagov/repos?per_page=100")
```

I know that users and orgs are functionally equivalent - the API returns the same thing - but pulling `alphagov` org data makes more sense to my brain.

Reply:
Don't think we can do that - this before block is just stubbing the endpoint that gets called by the Octokit SDK.

Reply:
👍🏻